TestAssets/TestProjects/AppWithLibrary/TestApp/TestApp.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" />
+  <Import Project="$(MSBuildSDKsPath)\microsoft.net.sdk\Sdk\Sdk.props" />
```

Comment:
This is case insensitive, right? Still we should spell it Sdks for consistency and conformance with FDG.